about.html

```diff
@@ -0,0 +1,127 @@
+          data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false"
+          aria-label="Toggle navigation">
+          <span class="toggler-icon top-bar"></span>
+          <span class="toggler-icon middle-bar"></span>
+          <span class="toggler-icon bottom-bar"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav nav">
+            <li class="nav-item">
+              <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Program</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Speakers</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+  </div>
+
+  <section class="container-fluid hero-section about-background about-hero-section">
+    <div>
+      <h6>Hi! Google fans and welcome to</h6>
+      <h1>Google I/O Conference 2022</h1>
+      <p class="pr-3">
+        Connect with developers from around the world for thoughtful discussions,
+        learn from Google experts, and get a first
+        look at our latest developer products.
+      </p>
+    </div>
+    <p class="container text-center mb-0">Please contact us for any quesions, suggestions or concerns.</p>
+    <p class="container text-center" id="email">google@gmail.com</p>
+  </section>
+
+  <section class="container-fluid about-global-io">
+    <h1 class="container text-center">Google I/O global conference 2022</h1>
+    <p class="text-center dash">____<p>
+    <div class="container about-logo-container">
+      <p class="text-center">
+        Connect with developers from around the world for thoughtful discussions,
+        learn from Google experts, and get a first
+        look at our latest developer products.
+      </p>
+      <img class="img-fluid google-io-logo mt-3 text-center" src="assets/images/google-io.jpeg" alt="google i/o logo"/>
+    </div>
+  </section>
+
+  <section class="container-fluid past-conferences">
+    <h1 class="container text-center mt-4">See the past Google I/O conferences</h1>
+    <p class="text-center dash">____<p>
+    <p class="text-center">
+      Take a look at the last Google I/O conferences that took place in Moutain view 
+      and San Francisco.
+    </p>
+    <div class="container past-google-ios">
+      <div class="col-md-6">
+        <img src="assets/images/google-io-2021.jpg" class="img-fluid" alt="google-io image" />
+      </div>
+      <div class="col-md-6">
+        <img  id="sundar-io" src="assets/images/google-io-2020.jpeg" class="img-fluid" alt="google-io image">
+      </div>
+    </div>
+  </section>
+
+  <!--  partner section -->
+  <div class="container-fluid partners-container">
+    <h1 class="text-center mb-4">Partners</h1>
+    <div class="container d-flex justify-content-around partners-1">
```

Comment:
Kindly go through this file, replace h1 tags, if there are others left, with smaller headers, as you see fit.